wallet-new/src/Cardano/Wallet/Kernel/DB/AcidState.hs

```diff
@@ -29,9 +29,11 @@ module Cardano.Wallet.Kernel.DB.AcidState (
   , UpdateHdWallet(..)
   , UpdateHdRootPassword(..)
   , UpdateHdAccountName(..)
+  , ResetAllHdWalletAccounts(..)
     -- *** DELETE
   , DeleteHdRoot(..)
   , DeleteHdAccount(..)
+  , DeleteAllHdAccounts(..)
```

Comment:
The paranoid guy in me ponders: should be expose such acidified function to the public? If we don't expect the users of the library to remove all the accounts, it might be good to not leave room for such temptation 😉

Reply:
I think this is actually dead code anyway from a previous version of this PR, so I can get rid of it.